system/jlib/jfile.cpp

```diff
@@ -6239,6 +6239,13 @@ class CMemoryMappedFile: implements IMemoryMappedFile, public CInterface
         CloseHandle(hfile);
```

Comment:
Just to be consistent, for WIN32, is UnmapViewOfFile() needed here or is it enough to CloseHandle(hmap) ?

Reply:
@ghalliday Can you comment?

Reply:
Good thinking, I think you are correct .  From the documentation:

"Mapped views of a file mapping object maintain internal references to the object, and a file mapping object does not close until all references to it are released. Therefore, to fully close a file mapping object, an application must unmap all mapped views of the file mapping object by calling UnmapViewOfFile and close the file mapping object handle by calling CloseHandle. These functions can be called in any order."

the same goes for the existing code in reinit() on line 6308.

@wangkx